src/main/java/com/esri/samples/na/find_service_areas_for_multiple_facilities/FindServiceAreasForMultipleFacilitiesSample.java

```diff
@@ -61,14 +73,7 @@ public void start(Stage stage) {
     stage.setScene(scene);
     stage.show();
 
-    // create control panel
-    VBox controlsVBox = new VBox(6);
-    controlsVBox.setBackground(new Background(new BackgroundFill(Paint.valueOf("rgba(0,0,0,0.3)"), CornerRadii.EMPTY,
-            Insets.EMPTY)));
-    controlsVBox.setPadding(new Insets(10.0));
-    controlsVBox.setMaxSize(150, 50);
-
-    // create buttons
+    // create button
```

Comment:
for set title a few lines above: shouldn't the title be "Find Service Area for Multiple Facilities Sample"?